tailwind.config.ts

```diff
@@ -1,2 +1,20 @@
-// this file is just here for intellisense
-// see: https://github.com/tw-in-js/vscode-twind-intellisense/issues/24#issuecomment-1670633807
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+  ],
+  theme: {
+    extend: {
+      backgroundImage: {
+        "gradient-radial": "radial-gradient(var(--tw-gradient-stops))",
+        "gradient-conic":
+          "conic-gradient(from 180deg at 50% 50%, var(--tw-gradient-stops))",
+      },
+    },
+  },
```

Comment:
this is a leftover from the shitty next boilerplate :tm:
```suggestion
  theme: {
    extend: {},
  },
```